packages/block-editor/src/components/block-list/index.js

```diff
@@ -86,6 +86,7 @@ function BlockList( {
 				'block-editor-block-list__layout',
 				className
 			) }
+			hasPopover={ __experimentalUIParts.hasPopover }
```

Comment:
This will cause a warning to be logged for any containers which are not the root container:

```
react-dom.8b3dda97.js?ver=16.9.0:539 Warning: React does not recognize the `hasPopover` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `haspopover` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

`Container` is assigned as a `div` for all but the root block list. `hasPopover` is not a valid attribute for a `div`.

Issue: #20069

Reply:
Issue now fixed with PR https://github.com/WordPress/gutenberg/pull/20082.